src/controllers/dashboard/plugins/add/index.js

```diff
@@ -22,6 +22,10 @@ function populateHistory(packageInfo, page) {
 function populateVersions(packageInfo, page, installedPlugin) {
     let html = '';
 
+    packageInfo.versions.sort((a, b) => {
+        return b.timestamp < a.timestamp ? 1 : -1;
+    });
+
```

Comment:
I see versions like this:
```
|Reports |v|
9.0.0.0
10.0.0.0
11.0.0.0 <-- selected
```
Everything is fine. But wouldn't it be better if versions go in reverse order?
```
|Reports |v|
11.0.0.0 <-- selected
10.0.0.0
9.0.0.0
```

Reply:
I suppose that might be useful, do you have an idea about the code changes? I can't test at the moment so it might not make 10.7.0 if we wait for me to find a desktop.

Reply:
> I suppose that might be useful, do you have an idea about the code changes?

See below.

We also need to check `targetAbi`. Someday :wink: